plugins/nsmserver/nsmserver_client.go

```diff
@@ -322,13 +418,15 @@ func startClientServer(id string, endpoints *nsmClientEndpoints) {
 	}
 
 	unix.Umask(socketMask)
-	sock, err := net.Listen("unix", listenEndpoint)
+
+	sock, err := newCustomListener(listenEndpoint) // net.Listen("unix", listenEndpoint)
```

Comment:
Any reason we are keeping this comment around?

Reply:
will remove, thanks for catching it.